src/devdb/devdb.py

```diff
@@ -43,3 +46,5 @@ def devdb():
     )()
 
     FlaggedJobsReport(data=data, qaqc_check_dict=QAQC_CHECK_DICTIONARY)()
+
+    FieldDistributionReport(data=data)()
```

Comment:
I would put the flagged jobs report at the bottom as it's the least pretty

Reply:
sounds good